src/SprykerEco/Yves/Computop/Theme/default/components/molecules/paypal-buttons/paypal-buttons.twig

```diff
@@ -0,0 +1,32 @@
+{% extends model('component') %}
+
+{% define config = {
+    name: 'paypal-buttons',
+    tag: 'paypal-buttons',
+} %}
+
+{% define data = {
+    clientId: required,
+    currency: required,
+    scriptSrc: 'https://www.paypal.com/sdk/js?client-id=' ~ data.clientId ~ '&currency=' ~ data.currency ~ '&intent=authorize',
+} %}
+
+{% define attributes = {
+    'order-data-url': '/computop/pay-pal-express-prepare',
+} %}
+
+{% block body %}
+    <div class="{{ config.jsName }}__buttons-container"></div>
+
+    {% include molecule('script-loader') with {
+        class: config.jsName ~ '__script-loader',
+        attributes: {
+            src: data.scriptSrc,
+        },
+    } only %}
+
+    {% include molecule('ajax-loader') with {
+        class: config.jsName ~ '__ajax-loader',
+        modifiers: ['big'],
+    } only %}
```

Comment:
- Please add blocks for these components.
- Add a variable for the `ajax-loader` modifier.